docs/components/concepts/messages.md

```diff
@@ -19,13 +19,21 @@ When a message is published and the message name and correlation key match to a
 A subscription is closed when the corresponding element (e.g. the message catch event), or its scope is left. After a subscription is opened, it is not updated (for example, when the referenced process instance variable is changed.)
 
 <details>
-   <summary>Publish message via zbctl</summary>
+   <summary>Publish message via Camunda 8 REST API</summary>
    <p>
 
 ```
```

Comment:
I could not seem to get cURL syntax highlights on a code block. Is there another suggestion? Should it work?